spa/src/system.ts

```diff
@@ -0,0 +1,33 @@
+import "import-map-overrides/dist/import-map-overrides";
+import "systemjs/dist/system";
+import "systemjs/dist/extras/amd";
+import "systemjs/dist/extras/named-exports";
+import "systemjs/dist/extras/named-register";
+import "systemjs/dist/extras/use-default";
+
+/**
+ * Gets the loaded import maps from SystemJS.
+ */
+export function getImportMaps() {
+  return Object.getOwnPropertySymbols(System)
+    .map((sym) => System[sym])
+    .filter((m) => typeof m.imports !== "undefined")
+    .map((m) => m.imports)
+    .pop();
```

Comment:
This seems like a hack. If that's the way to do it, that's fine, but I figure this might be the most fragile piece of this otherwise quite solid-seeming initial script.

Reply:
Yes, definitely. Unfortunately, there is no way of doing this using the official SystemJS APIs.